src/jetstream/auth.go

```diff
@@ -430,12 +431,20 @@ func (p *portalProxy) DoLoginToCNSI(c echo.Context, cnsiGUID string, systemShare
 				Admin:       isAdmin,
 			}
 
+			// TODO: Only do this for OAuth token types
```

Comment:
Should this be addressed in this PR? Otherwise should be a reference to a ticket

Reply:
The comment needs to be more descriptive. The following line fetches the user for the token. This could be any type of token (OAuth or otherwise). Why is it limited to OAuth only?